entrypoint.sh

```diff
@@ -24,49 +24,29 @@
 #
 # Entrypoint for CMS Ansible Execution Environment (aee)
 
-PREVIOUS_LAYER_DIR=/inventory/layer${LAYER_PREVIOUS}
-CURRENT_LAYER_DIR=/inventory/layer${LAYER_CURRENT}
-
-create_complete() {
-  mkdir -p ${CURRENT_LAYER_DIR}
-  echo $1 > ${CURRENT_LAYER_DIR}/complete
-}
-
-wait_for_previous() {
-  PREVIOUS_FLAG=${PREVIOUS_LAYER_DIR}/complete
-  until [ -f ${PREVIOUS_FLAG} ]
-  do
-       sleep 5
-       echo "Waiting for the previous configuration layer to complete"
-  done
-  echo "Previous layer completed"
-  COMPLETE=$(cat $PREVIOUS_FLAG)
-  if [ $COMPLETE -ne 0 ]; then
-      echo "The previous configuration layer encountered an error, so this layer will not be run.";
-  fi
-  return $COMPLETE
-}
-
 /entrypoint-setup.sh
 
 if [ $? -ne 0 ]; then
-    create_complete 1
-    exit;
-fi
-
-if [ ! -z ${LAYER_PREVIOUS:+x} ]; then
-  wait_for_previous
-fi
-
-if [ $? -ne 0 ]; then
-    create_complete 1
-    exit;
+    exit 1;
 fi
 
-echo "Running $SESSION_PLAYBOOK from repo $SESSION_CLONE_URL"
-
-"$@" 2>&1
-ANSIBLE_EXIT=$?
-
-create_complete $ANSIBLE_EXIT
-exit $ANSIBLE_EXIT
+ANSIBLE_DIR=/etc/ansible
+
+for layer in $(echo "${@}" | jq -c .[]); do
+    export SESSION_CLONE_URL=$(echo "${layer}" | jq -r .cloneUrl)
+    export SESSION_PLAYBOOK=$(echo "${layer}" | jq -r .playbook)
+    export LAYER_CURRENT=$(echo "${layer}" | jq -r .layer)
+    LAYER_DIR=${ANSIBLE_DIR}/layer${LAYER_CURRENT}
```

Comment:
Two things:
* I feel that including a dash or underscore would better distinguish the text 'layer' and ${LAYER_CURRENT}?
** Something like <ANS_DIR>/layer-<foo>
* What happens if any of these attributes isn't found? Especially with LAYER_CURRENT, you could end up clobbering all of the layers where .layer doesn't evaluate to anything.

Reply:
1) This has been the naming scheme since we added multiple layers.  I don't really want to mess with this right now.
2) The data used here is passed in by the cfs-operator.  The first two fields are straight from the configuration info and the layer is added to the data before it's passed to this container.  I didn't worry about validating this data because there's not really a way for it to be invalid unless you created an AEE container manually, but that's never done and has so many dependencies it's not worth worrying about.  I can make this fail explicitly if you like, but the truth is if there's a problem with the fields there's probably bigger issues and this is going to fail anyway.